shared/src/test/diff/nu/UserDefinedAnnotations.mls

```diff
@@ -14,9 +14,48 @@ fun foo = 0
 class tailrec2() extends Annotation
 //│ class tailrec2() extends Annotation
 
-// :e // TODO should be rejected
+:e // In the future, this should be supported.
```

Comment:
My bad, I should've stated it more clearly. I meant that we should add support for annotations with parameters in the future.

Reply:
Got it. Then why not add a new test case using `@tailrec2()` and move the comment to that one?

Reply:
Perhaps I should just leave it in a comment, and not as an actual test. Such a test case would actually pass, because it will add a `tailrec2` annotation to the unit literal `()`, which feels wrong. I'll need to make some non-trivial changes to the parser to implement this in the end.